portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,33 @@
 <!DOCTYPE html>
 <html>
```

Comment:
Indent by 2 spaces at a time (this file and others)
https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
I think if you run the prettier it should automatically fix this for you